felix/bpf/ut/midflow_test.go

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) 2019-2021 Tigera, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ut_test
+
+import (
+	"net"
+	"testing"
+
+	"github.com/google/gopacket/layers"
+	. "github.com/onsi/gomega"
+
+	//tcdefs "github.com/projectcalico/calico/felix/bpf/tc/defs"
+	"github.com/sirupsen/logrus"
+
+	"github.com/projectcalico/calico/felix/bpf/nat"
+)
+
+func TestMidflowFailoverNoConntrack(t *testing.T) {
```

Comment:
Just one note, prepend the test with Maglev and also the file so that we know what specificaly it applies to

Reply:
Thanks - BTW this PR is old.
Latest work is on github.com/aaaaaaaalex/calico/pull/1